src/test/java/com/selfxdsd/todocli/TodoIDTest.java

```diff
@@ -0,0 +1,75 @@
+package com.selfxdsd.todocli;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TodoIDTest {
```

Comment:
@lukacupic make this class ``final`` and add JavaDocs to it and also to the tests please.